run_scripts/start_bag.il

```diff
@@ -1256,6 +1265,14 @@ procedure( create_layout_helper( cv tech_file inst_list rect_list via_list pin_l
                         obj = dbCreateParamSimpleMosaic(cv inst_cv inst->name inst->loc inst->orient
                                                         inst->num_rows inst->num_cols inst->sp_rows inst->sp_cols
                                                         inst->params)
+                        when( obj
+                            if( inst->param_order != nil then
+                                param_order=inst->param_order
+                            else
+                                param_order= mapcar( lambda( (x) car(x) ) inst->params)
+                            )
+                            CCSinvokeInstCdfCallbacks(obj ?order param_order)
```

Comment:
This method of calling the callbacks seems to be a lot faster. The only difference from the master branch seems to be that the method used for callbacks is changed from CCSinvokeCdfCallbacks() to CCSinvokeInstCdfCallbacks(). This seems to work on our side (e.g. layouts get updated correctly AND generation speed is much faster).